doc/project/hw_stages.md

```diff
@@ -177,6 +177,22 @@ The commit ID has its own entry in the project Hjson file, as shown below.
 }
 ```
 
+### Other optional fields
+
+Additionally, the tool that generates the dashboard accepts the following optional fields: the design specification, the DV plan and the checklist.
+They are set as partial paths (relative references) to the respective documents as shown below.
```

Comment:
"relative" typically means relative to where you are in the current directory
structure. I recommend "(reference relative to top of repository)"